src/table/cells/withSelections.jsx

```diff
@@ -1,12 +1,19 @@
 import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
+import { handleCellFocus } from './handle-cell-focus';
 import { selectCell } from '../selections-utils';
 import { getSelectionStyle } from '../styling-utils';
 
 export default function withSelections(CellComponent) {
   const HOC = (props) => {
-    const { cell, selState, selDispatch, styling } = props;
-    const handleMouseUp = (evt) => cell.isDim && evt.button === 0 && selectCell(cell, selState, selDispatch, evt);
+    const { cell, selState, selDispatch, styling, focusedCell, setFocusedCell, rootElement } = props;
+    const handleMouseUp = (evt) => {
+      cell.isDim &&
+        evt.button === 0 &&
+        handleCellFocus(cell, focusedCell, setFocusedCell, rootElement) &&
```

Comment:
I might have mentioned this already, but we might need to move this outside selections later. So you can click to focus but not make selections. This is fine for now though :)

Reply:
click to focus but not make selections, in the real scenario, does such a case exist?

Reply:
It does, try the old table in edit mode and click a cell